modules/ROOT/pages/security/ssl-framework.adoc

```diff
@@ -13,138 +13,76 @@ The SSL framework provides support for securing the following Neo4j communicatio
 This page describes how to set up SSL within your environment, how to view, validate, and test the certificates.
 
 [[ssl-providers]]
-== SSL providers
+== SSL Providers
```

Comment:
```suggestion
== SSL providers
```

Reply:
oh I was going off the documentation style guide that says only capitalise proper nouns and the first word of sentences/titles. https://development.neo4j.dev/docs/docs-style-guide/general-rules/#_capitalization